spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc

```diff
@@ -3,330 +3,379 @@
-be clickable and on pressing it the UI will proceed to create each task definition. If there
-are any errors during creation then after creation finishes the
-editor will show any lines of input, as it cannot be used in task definitions.
-These can then be fixed up and creation repeated.  There is an
-*import file* button to open a file browser on the
-local file system if the definitions are in a file and it is easier
-to import than copy/paste.
+After entering any data, a validator runs asynchronously to verify the syntax, that the application name entered is a valid application, and that it supports the options specified.
+If validation fails, the editor shows the errors with more information available in tooltips.
+
+To make it easier to enter definitions into the text area, content assist is supported.
+Pressing Ctrl+Space invokes content assist to suggest simple task names (based on the line on which it is invoked), task applications, and task application options.
+Press Escape to close the content assist window without choosing a selection.
+
+If the validator should not verify the applications or the options (for example, when specifying non-whitelisted options to the applications), you can turn off that part of validation by toggling the checkbox off on the Verify Apps button.
+The validator then performs only syntax checking.
+When an app is correctly validated, the Create button becomes clickable.
+Clicking it causes the creation of each task definition.
+If there are any errors during creation, then, after creation finishes, the editor shows any lines of input that cannot be used in task definitions.
+These can then be fixed, and the creation process can be repeated.
+The Import File button opens a file browser on the local file system, for those cases where the definitions are in a file and it is easier to import than copy and paste.
 
 NOTE: Bulk loading of composed task definitions is not currently supported.
 
+
+
 ==== Creating Composed Task Definitions
-The dashboard includes the Create Composed Task tab that provides the canvas
-application, offering a interactive graphical interface for creating
-composed tasks.
+
+The dashboard includes the Create Composed Task tab, which provides an interactive graphical interface for creating composed tasks.
 
 In this tab, you can:
 
-* Create and visualize composed tasks using DSL, a graphical canvas, or both
-* Use auto-adjustment and grid-layout capabilities in the GUI for simpler and
-interactive organization of the composed task
+* Create and visualize composed tasks using DSL, a graphical canvas, or both.
+* Use auto-adjustment and grid-layout capabilities in the GUI for simpler and interactive organization of the composed task.
+
+On the Create Composed Task screen, you can define one or more task parameters by entering both the parameter key and the parameter value.
+
+NOTE: Task parameters are not typed.
+
+The following image shows the composed task designer:
```

Comment:
The composed-task images are old-UI screenshots. It would be better to replace them for consistency.

Reply:
made an issue to track